tests/sles4sap/saptune/mr_test.pm

```diff
@@ -35,7 +35,7 @@ sub setup {
     assert_script_run "echo 'export PATH=\$PATH:\$HOME' >> /root/.bashrc";
     # Remove any configuration set by sapconf
     assert_script_run "sed -i.bak '/^@/,\$d' /etc/security/limits.conf";
-    assert_script_run 'mv /etc/systemd/logind.conf.d/sap.conf{,.bak}';
+    assert_script_run "mv /etc/systemd/logind.conf.d/sap.conf{,.bak}" unless check_var('DESKTOP', 'textmode');
```

Comment:
```suggestion
    assert_script_run 'mv -f /etc/systemd/logind.conf.d/sap.conf{,.bak}';
```
With the `-f` option, `mv` will not fail if the sap.conf file doesn't exist.

Reply:
No I tested it :) But I thought too.